lib/foods.js

```diff
@@ -8,7 +8,9 @@ $(document).ready(function() {
   $('.food-options').on('click', '.food-delete', () => {
     handleDeletingFood();
   });
-
+  $('.food-options').on('click', '.food-name, .food-calories', () => {
```

Comment:
🥇 @case-eee & @TheKaterTot there are two small pieces of functionality missing here, and I would appreciate any thoughts/suggestions either of you have in order to achieve them:
1) On a click, it creates an input field for the user to update either a name or calorie for a food. However, the user has to click into the field in order to start typing. Is there a way to automatically place a cursor into the input field?
2) The new value saves to the DOM and is updated in the DB when the user clicks out of field -- using enter to submit does not yet work, is the best way to approach it to create another event listener for an enter key press? 

Thank you! 🙋